src/backend/distributed/executor/repartition_join_execution.c

```diff
@@ -127,11 +123,11 @@ static char *
 GenerateCreateSchemasCommand(List *jobIds, char *ownerName)
 {
 	StringInfo createSchemaCommand = makeStringInfo();
-	ListCell *jobIdCell = NULL;
 
-	foreach(jobIdCell, jobIds)
+	void *jobIdPointer = NULL;
```

Comment:
So noticed this while converting. Elsewhere we `palloc(sizeof(uint64))` but in this instance the decision was made to convert between `uint64` & `void*`

If we're going to require 64-bit systems, we should make sure we refuse to compile on such targets & might as well update code elsewhere to rely on being able to losslessly convert between `uint64` & `void*`. Otherwise this should be updated to `uint64*`